samples/metrics_monitor/src/cttmetrics_i915_pmu.cpp

```diff
@@ -40,12 +40,38 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <time.h>
 #include <unistd.h>
 #include <i915_drm.h>
+#include <xf86drm.h>
+
+char I915_DRI_DIR[16] = "/dev/dri/card0";
```

Comment:
This will limit the usage to the 1st card being intel and limit access to privileged user. Access to render node would be more preferable I think. You may try to explore libdrm for the helper function to open the device.

Reply:
Addressed